network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -584,6 +622,19 @@ def disjoint_referenced_ip_blocks(self):
         return IpBlock.disjoint_ip_blocks(self.config1.get_referenced_ip_blocks(),
                                           self.config2.get_referenced_ip_blocks())
 
+    @staticmethod
+    def clone_without_ingress(config):
+        """
+        Clone config without ingress policies
+        :param config: the config to clone
+        :return: resulting config without ingress policies
+        """
+        config_without_ingress = config.clone_without_policies(config.name)
+        for policy in config.policies.values():
+            if not isinstance(policy, IngressPolicy):  # ignoring ingress policies
+                config_without_ingress.append_policy_to_config(policy)
+        return config_without_ingress
+
```

Comment:
Maybe would be more efficient to first check if Ingress policies exist in the config, and if not - just return the input config..

Reply:
Done